ssc/Pos/Ssc/Extra/Logic.hs

```diff
@@ -235,9 +235,6 @@ sscApplyBlocksFinish
     => SscGlobalState ssc -> m [SomeBatchOp]
 sscApplyBlocksFinish gs = do
     sscRunGlobalUpdate (put gs)
-    inAssertMode $
-        logDebug $
-        sformat ("After applying blocks SSC global state is:\n"%build) gs
```

Comment:
Why did you delete it? It's only in assert mode and with debug severity. It's sometimes useful.